packages/stencil/src/components/datacom-avataravatar/readme.md

```diff
@@ -0,0 +1,17 @@
+# datacom-avataravatar
```

Comment:
Not related to this file, but other components have an additional 'usage.md' file that the devs have written to explain how the component is meant to be used. Should probably write one for this component too (and feel free to check the other usage.md files for examples of how to format.)